sdk/go/common/workspace/paths.go

```diff
@@ -125,9 +125,17 @@ func DetectProjectStackPath(stackName tokens.QName) (string, error) {
 // DetectProjectPathFrom locates the closest project from the given path, searching "upwards" in the directory
 // hierarchy.  If no project is found, an empty path is returned.
 func DetectProjectPathFrom(path string) (string, error) {
-	return fsutil.WalkUp(path, isProject, func(s string) bool {
+	path, err := fsutil.WalkUp(path, isProject, func(s string) bool {
```

Comment:
this doesn't return an error and is intended not to. Handle the error in the caller since the caller is incorrectly assuming errors if not found.